fgd/point/math/math_counter.fgd

```diff
@@ -28,6 +28,6 @@
 	output OnHitMin(void) : "Fired when the counter value meets or goes below the min value. The counter must go back above the min value before the output will fire again."
 	output OnHitMax(void) : "Fired when the counter value meets or exceeds the max value. The counter must go below the max value before the output will fire again."
 	output OnGetValue(float) : "Fired in response to the GetValue input. Used for polling the counter when you don't want constant updates from the OutValue output."
-	output OnChangedFromMin[since_ASW, MBase](void) : "Fired when the counter value changes from the minimum value."
-	output OnChangedFromMax[since_ASW, MBase](void) : "Fired when the counter value changes from the max value."
```

Comment:
And same here. I won't do it for every instance (it's a big boi of a PR) so I'll leave it to you to spot the remaining ones.